motor_controller/test/unittest_motor_controller.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import rosunit
+from motor_controller.motor_controller import MotorController
+from unittest import TestCase, mock
+
+
+#@patch('motor_controller.motor_controller.serial.Serial')
+class TestMotorController(TestCase):
+
+    def setUp(self):
+        self.patcher = mock.patch('motor_controller.motor_controller.serial.Serial')
+        self.mock_serial = self.patcher.start().return_value
+        self.sc = MotorController()
```

Comment:
Does this even work without the motor controller plugged in? Since you don't mock the ports it would not be able to find a polulu port without it being plugged in.

Reply:
Correct, this was a oversight after fixing the dynamic allocation of ports. Will fix this.

Reply:
Fixed.